src/bases/createStateful.ts

```diff
@@ -104,6 +104,14 @@ const createStateful: StatefulFactory = createEvented
 	.mixin({
 		className: 'Stateful',
 		mixin: {
+			get stateFrom(this: Stateful<State>): ObservableState<State> | undefined {
+				const observedState = observedStateMap.get(this);
+				if (observedState) {
+					return observedState.observable;
+				}
+				return undefined;
```

Comment:
Isn't this implied?  Or was TypeScript just being awkward? Or did you just want to be explicit?